include/deal.II/base/bounding_box.h

```diff
@@ -108,7 +108,7 @@ BoundingBox<spacedim, Number>::BoundingBox (const std::pair<Point<spacedim,Numbe
 {
   //We check the Bounding Box is not degenerate
   for (unsigned int i=0; i<spacedim; ++i)
-    Assert (boundary_points.first[i] < boundary_points.second[i],
+    Assert (boundary_points.first[i] <= boundary_points.second[i],
```

Comment:
These days I've been working hard on my code an I lost this, sorry!!!

So, I have no problem with creating an empty bounding box like this one: the Assert is there because, as far as I'm concerned, if I'm creating an empty box "manually" by writing the single points then I'm doing something wrong, probably...
If you need to create one and then use compute_bounding_box then you could avoid altogether using the constructor with separate points; but if you have some good reason to allow building by hand an empty box than I have no problem...